### PR TITLE
PP-68 Delete class

### PR DESCRIPTION
**Pull request by vrudnitskaya**
### Description
The functionality for deleting the class has been implemented. 

Also added is the display of available lesson time in the teacher's classes.

Adjusted the visibility of the heading in notifications depending on the content.
### Screenshots
delete-modal
Error message
error

Class information
class-info

no-info

### CODE REVIEW

File: src/util/DeleteClassService.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+const useDeleteClass = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { userData, setUserData } = useAuth();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const openModal = () => setIsModalOpen(true);
+  const closeModal = () => setIsModalOpen(false);
+
+  const handleDelete = async (token, classId, onRequestClose, onError) => {
```
Comment (mariyador) [inline src/util/DeleteClassService.js:14]: If I have only 1 class and delete it receive the info about the class that i hav just deleted> but i have to get a page with with button create your first class

Comment (vrudnitskaya) [reply src/util/DeleteClassService.js:14]: got it. I'll fix it.

Comment (vrudnitskaya) [reply src/util/DeleteClassService.js]: fixed it.

File: src/util/DeleteClassService.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  const [isLoading, setIsLoading] = useState(false);
+
+  const openModal = () => setIsModalOpen(true);
+  const closeModal = () => setIsModalOpen(false);
+
+  const handleDelete = async (token, classId, onRequestClose, onError, onSelectedClass, onSelectedId) => {
+    try {
+        setIsLoading(true);
+      let response = await deleteClass(token, classId);
+      if (response.status === 200) {
```
Comment (JoHeyJo) [inline src/util/DeleteClassService.js:18]: You are handling an error correctly here but what if you get a non 200 response. That's not technically an error and also your loading state would be stuck here. You just need an else conditional to handle that non 200 response.

**Review by JoHeyJo — APPROVED** [review]
This looks pretty good. Nicely done! Just take a look at the comment but otherwise this looks good. 

Stretch Goal: If a student is signed up for a class and it gets deleted by the teacher. The student should get a notification that the class has been removed.

**Comment by vrudnitskaya** [comment]
> This looks pretty good. Nicely done! Just take a look at the comment but otherwise this looks good.
> 
> Stretch Goal: If a student is signed up for a class and it gets deleted by the teacher. The student should get a notification that the class has been removed.

File: src/util/DeleteClassService.js (hunk 2 above)
Comment (vrudnitskaya) [reply src/util/DeleteClassService.js:18]: got it.

**Review by ofeterovskaya — APPROVED** [review]
Thank you Valya, Nice job! You have conflicting files, please resolve the conflicts before you merge this branch

**Review by mariyador — APPROVED** [review]
Thank you, Valya!

**Pull request merged**